components/avatar.css

```diff
@@ -0,0 +1,51 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container_avatar{
+    margin: auto;
+    padding:1rem;
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-end;
+}
+
+.avatar_circle .avatar{
+    border-radius: 50%;
+}
+
+.avatar_round_corner .avatar{
+    border-radius: .5rem;
```

Comment:
for values less than 1rem you can use px directly as suggested by Vikram Sir, rest is fine 👍